lib/resultsCapturer.js

```diff
@@ -58,15 +58,13 @@ module.exports = function () {
   });
 
   this.registerHandler('StepResult', function (event, callback) {
```

Comment:
https://github.com/cucumber/cucumber-js/blob/v1.1.0/lib/cucumber/runtime/event.js the message says that the argument is now the associated object not the event, so I think it makes sense to name it appropriately. Since we know what handler we're registering, then we know the type of the argument.

This is what I mean:
`registerHandler('StepResult', function (stepResult, callback) {`
`registerHandler('BeforeFeature', function (feature, callback) {`
`registerHandler('AfterScenario', function (scenario, callback) {`